python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -26,16 +26,16 @@
 
 import abc
 import itertools
+import logging
 
 import astropy.time
 import astropy.units
 import numpy
 
 import lsst.geom as geom
-import lsst.afw.table as afwTable
+import lsst.afw.table
```

Comment:
I did it because in that file 2/3 of the code used lsst.afw.table and the other third used afwTable. I made them consistent (and did it on a distinct commit) by picking the most common variant.

Reply:
But now it is inconsistent with the other module imports in the file, I could be mistaken on this, but I do think the dev guide asks us to stay consistent w/n a file.

Reply:
It's more consistent than it was but I'll make it consistent the other way.

Reply:
I guess I am not being clear, yes it is more consistent for `lsst.afw.table` which seems like it was only working as a side effect anyway. I don't have a problem if you choose to change it to that. However, `lsst.afw.table` is now not at all consistent with `pexConfig`, `pipeBase`, `geom`. So if you want to fix everything so that the full import path is preferred in this file, change the other imports as well.

Reply:
Fixed. You are right that this does look better.